src/app/file-manager/services/electron-ipc.api.service.ts

```diff
@@ -38,5 +40,35 @@ export class ElectronIpcApiService extends ElectronIpcService {
   public logout(): void {
     this.electronService.ipcRenderer.send(constants.IPC_GOOGLE_LOGOUT);
   }
-}
 
+  public exportS3(dataTransfer: any): void {
+    this.electronService.ipcRenderer.removeAllListeners(constants.IPC_EXPORT_S3);
```

Comment:
I think this came up before, did we try using `once()` to avoid having to use `removeAllListeners()`?

Reply:
Even more, I think we don't need to remove listeners here.